fontbe/src/avar.rs

```diff
@@ -21,7 +22,30 @@ pub fn create_avar_work() -> Box<BeWork> {
     Box::new(AvarWork {})
 }
 
-fn to_segment_map(axis: &Axis) -> Option<SegmentMaps> {
+/// Return true if the SegmentMaps is not a boring identity map
+fn is_interesting(segment_map: &SegmentMaps) -> bool {
+    segment_map
+        .axis_value_maps
+        .iter()
+        .any(|av| av.from_coordinate != av.to_coordinate)
+}
```

Comment:
I think this makes sense as a method on the `SegmentMaps` type in write-fonts? I'd probably flip the logic and call it `is_identity`.


One option would be to add the method there, merge this as-is, and open an issue to fix this when we next publish write-fonts.

Reply:
+1 making the method is_identity